driver/config/.schema/config.schema.json

```diff
@@ -536,13 +581,7 @@
                   "default": "https://www.ory.sh/kratos/docs/fallback/verification"
                 },
                 "after": {
-                  "type": "object",
-                  "properties": {
-                    "default_browser_return_url": {
-                      "$ref": "#/definitions/defaultReturnTo"
-                    }
-                  },
-                  "additionalProperties": false
+                  "$ref": "#/definitions/selfServiceAfterVerification"
```

Comment:
This change is causing schema tests to fail. You would need to change `test/schema/fixtures/config.schema.test.success/root.full.yml` to:
```yaml
 verification:
      enabled: true
      ui_url: /verification
      lifespan: 1h
      after: "#/definitions/selfServiceAfterVerification"
```

Reply:
PR for this https://github.com/innoq/kratos/pull/1

Reply:
Merged